sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -194,6 +196,55 @@ export class StudentService extends RecordDataModelService<Student> {
       throw error;
     }
 
+    // If sfasIndividual wasn't found, check for a partial match
+    if (!sfasIndividual) {
+      let partialMatch: SFASIndividual;
+      try {
+        partialMatch =
+          await this.sfasIndividualService.getIndividualStudentPartialMatch(
+            user.lastName,
+            student.birthDate,
+            studentSIN,
+          );
+
+        if (partialMatch) {
+          // Send out a notification of a partial match for the student information.
+          let matches = "";
+          if (
+            user.lastName === partialMatch.lastName &&
+            student.birthDate === partialMatch.birthDate
+          ) {
+            matches = "Last name and birth date match.";
+          } else if (
+            user.lastName === partialMatch.lastName &&
+            studentSIN === partialMatch.sin
+          ) {
+            matches = "Last name and SIN match.";
+          } else if (
+            studentSIN === partialMatch.sin &&
+            student.birthDate === partialMatch.birthDate
+          ) {
+            matches = "Birth date and SIN match.";
+          }
```

Comment:
The `last name` checks must be case insensitive in the same way the SQL is also executing a case insensitive.

Reply:
updated